src/content/concepts/why-webpack

```diff
@@ -0,0 +1,55 @@
+---
+title: Why webpack
+sort: 7
+contributors:
+- debs-obrien
+---
+
+## Why webpack?
+
+To understand why you should use webpack lets do a recap of how we use JavaScript on the web. The origins of how we use JavaScript on the web. 
+
+JavaScript is just a script and there are 2 ways to use it in the browser, loaded as a script tag or written directly inside your html. The problems that this causes is that it doesn't scale. You either have too many scripts that you try to load from your script tags in the html and this causes a huge performance problem. There is only a certain amount of concurrent requests that can fetch data at a single time so breaking your application into a hundred JavaScript files and loading them in the browser is a huge performance problem. One big JS file results in unmaintainable scripts that causes problems in scope, size, readability, fragility and monolith files.
```

Comment:
I think the beginning is a little off, I would go with something more than:

> There are two ways to run JavaScript in a browser. First, include a script for each functionality you want to implement, the _issue_ is that the solution is hard to scale as loading too many scripts causes a network bottleneck.

Goo to mention first first, to use the word _issue_ instead of _problem_ when possible. Second, it's important to not say _performance_ when is actually a network anti-pattern, so we can differentiate other features like `ModuleConcatenation` from webpack that does actually improve performance of the script indeed.

I would start the final part with:

> The other alternative is load a big `.js` file containing all your project code, but this results in...

Reply:
ok cool, think i put this part in right but if i deleted too much let me know